eslint/README.md

```diff
@@ -0,0 +1,88 @@
+# [ESL](../../../) ESL migration support
+
+Version: *4.12.0*
+
+Authors: *Natalia Smirnova, Alexey Stsefanovich (ala'n)*.
+
+<a name="intro"></a>
+
+This article aims to assist the smooth migration process from older versions of ESL to the latest version. To support this transition, we have developed an ESLint plugin designed specifically for ESL library. This plugin targets deprecated features and aliases within the library, offering a seamless means of identifying these outdated elements. Additionally, it suggests suitable replacements, thereby ensuring a hassle-free upgrade to the newest version of ESL.
+
+### Installation
+To use custom ESLint plugin, you need to install it as a development dependency in your project. You can do this using npm:
+
+```bash
+npm install --save-dev @exadel/eslint-plugin-esl
+```
+
+### Usage
+To configure plugins inside of a configuration file, use the plugins key, which contains a list of plugin names:
+
+```json
+{
+  // ...
+  "plugins": [
+    "@exadel/esl"
+  ]
+  // ...
+}
+```
+
+Or in YAML:
+```js
+  plugins: 
+    - "@exadel/esl"
+```
+
+Once you have added the plugin to your project, you can define specific ESLint rules for handling ESL deprecations in your ESLint configuration file. Here are some examples of how to configure them:
```

Comment:
Id suggests reordering the usage description. 
We do not expect 99.9% of people will use it manually.
The main strategy is to include preset according to the plan of migration:
 - to keep consistent with esl updates but in light mode  - apply 4.0 preset, which is just warns you if you have any deprecated features in code
 - in order to be compatible with version 5.0.0 (which is coming by the end of this year) - apply 5.0 preset which is strictly deprecates the usage of the features that are going to be removed.